pkg/collect/run.go

```diff
@@ -103,7 +113,10 @@ func runWithoutTimeout(ctx context.Context, c *Collector, pod *corev1.Pod, runCo
 	limits := troubleshootv1beta2.LogLimits{
 		MaxLines: 10000,
 	}
-	podLogs, err := getPodLogs(ctx, client, *pod, runCollector.Name, "", &limits, true)
+	if runCollector.CollectorName == "" {
+		runCollector.CollectorName = fmt.Sprintf("run-collector-pod-%s", pod.Name)
+	}
+	podLogs, err := getPodLogs(ctx, client, *pod, runCollector.CollectorName, "", &limits, true)
```

Comment:
@manavellamnimble can we generate file name here and pass it to `getPodLogs`?  and then update other places where `getPodLogs` is called as well.   This will allow you to keep everything backwards compatible.